edi_oca/models/edi_exchange_record.py

```diff
@@ -108,6 +108,7 @@ class EDIExchangeRecord(models.Model):
         compute="_compute_retryable",
         help="The record state can be rolled back manually in case of failure.",
     )
+    company_id = fields.Many2one("res.company", string="Company", readonly=True)
```

Comment:
why readonly?

Reply:
I thought it was good idea to prevent users to accidentally or intentionally change the company associated with a record. I hadn't thought of any situation where someone would want to do that. But, nvm, I have changed it.

Reply:
@OriolMForgeFlow too late, it has been merged... :smile: 
I get your point, feel free to propose a change.
Also, regarding the default: feel free to propose a change as well :wink: